core/components/atoms/chip/__stories__/OverflowBehavior.story.jsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { Chip, Row, Column } from '@/index';
+
+export const overflowBehavior = () => {
+  const renderChips = (
+    <>
+      <Row>
```

Comment:
Use of React Fragment is redundant here. Please check this once

Reply:
done